src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -449,6 +433,36 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 	}
 
+	private loadContentMetadata(metadata: any): void {
+		this._savedKernelInfo = metadata.kernelspec;
+		this._defaultLanguageInfo = metadata.language_info;
+		// If language info was serialized in the notebook, attempt to use that to decrease time
+		// required until colorization occurs
+		if (this._defaultLanguageInfo) {
+			this.updateLanguageInfo(this._defaultLanguageInfo);
+		}
+		this._savedConnectionName = metadata.connection_name;
+		this._multiConnection = metadata.multi_connection ? metadata.multi_connection : false;
```

Comment:
undefined is "falsy", so you can shorten this with a double negation: !!metadata.multi-connection